### PR TITLE
Add github templates

### PR DESCRIPTION
**Pull request by rimusz**
Signed-off-by: rimas <rmocius@gmail.com>

Solves #46

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+**Version of Helm and Kubernetes**:
+
```
Comment (nvtkaszpir) [inline .github/ISSUE_TEMPLATE.md]: please add a comment like:
```
<!-- to get current helm and kubernetes versions try:
helm version
and paste output here.
-->
```

Comment (rimusz) [reply .github/ISSUE_TEMPLATE.md]: done

Comment (unguiculus) [reply .github/ISSUE_TEMPLATE.md]: Add ct's version.

`ct version`

Comment (rimusz) [reply .github/ISSUE_TEMPLATE.md]: done

**Pull request merged**